packages/terra-clinical-item-collection/src/_ItemCollectionUtils.jsx

```diff
@@ -0,0 +1,79 @@
+const KEYCODES = {
+  ENTER: 13,
+  SPACE: 32,
+};
+
+/**
+ * This function ensures the correct elements are provided to create a consistent item view layout. To handle a
+ * potential missing required accessory element, a boolean indicator is provided to single accessory space is needed.
+ *
+ * @ param {childElement} obj - The react child props that will compose an item view list item.
+ * @ param {requiredElements} obj - The elements expected to be displayed.
+ */
+function addAnyMissingListElements(childElements, requiredElements) {
```

Comment:
Nitpick:

I think Any is extraneous here. addMissingListElements seems sufficient enough imo